### PR TITLE
std::prelude: code consistency nits

### PR DESCRIPTION
**Pull request by tshepang**

### CODE REVIEW

File: src/libstd/prelude/v1.rs [hunk 1]
```diff
@@ -35,9 +34,7 @@
 #[stable(feature = "rust1", since = "1.0.0")]
 #[doc(no_inline)] pub use iter::DoubleEndedIterator;
 #[stable(feature = "rust1", since = "1.0.0")]
-#[doc(no_inline)] pub use iter::ExactSizeIterator;
-#[stable(feature = "rust1", since = "1.0.0")]
-#[doc(no_inline)] pub use iter::{Iterator, IteratorExt, Extend};
+#[doc(no_inline)] pub use iter::{Iterator, IteratorExt, Extend, ExactSizeIterator};
```
Comment (alexcrichton) [inline src/libstd/prelude/v1.rs]: This was done to keep the line under 80 characters, can you leave as it was before?

Comment (tshepang) [inline src/libstd/prelude/v1.rs]: I though it was fine since the guidelines allow 100 characters

Comment (alexcrichton) [inline src/libstd/prelude/v1.rs]: Yes but it is more important to stick within the existing style already present within a file.

Comment (tshepang) [inline src/libstd/prelude/v1.rs]: It's done

**Pull request merged**